gems/smithy/lib/smithy/anvil/client/templates/specs/endpoint_provider_spec.erb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+require_relative 'spec_helper'
+
+module <%= namespace %>
+  describe EndpointProvider do
+    subject { EndpointProvider.new }
+
+    <% test_cases.each do |test_case| %>
```

Comment:
Not apparent unless you inspect the generated code, but all ERB tags should have dashes <%- or -%> because of the trim mode that is selected in the base view class.